python/cross_service/resilient_service/README.md

```diff
@@ -0,0 +1,183 @@
+
+An Auto Scaling group starts Amazon EC2 instances in a specified set of Availability Zones. 
+This example uses an Auto Scaling group to ensure that the number of running instances 
+is within a specified range and across multiple Availability Zones. The Auto Scaling group
+is set as a load balancer target so that HTTP requests are handled by a single endpoint
+and dispatched equally to the instances in the group.
+
+An [AWS Identity and Access Management (IAM)](https://docs.aws.amazon.com/IAM/latest/UserGuide/introduction.html) 
+instance profile specifies permissions that are granted to Amazon EC2 instances created 
+during the demo. By associating an instance profile with an instance, you let AWS SDK code 
+that runs on the instance assume the profile's role to acquire the permissions specified
+by the role's attached policies.
+
+An Amazon EC2 launch template specifies how instances are created. This example creates
+a launch template that specifies the instance type, Amazon Machine Image (AMI), instance 
+profile, and a Bash script that is run when the instance is started. The Bash
+script installs required Python packages and starts a demo Python web server that listens
+for HTTP requests on port 80. The Python web server uses the SDK for Python to get 
+recommendation data from a DynamoDB table and to get parameter values from Systems 
+Manager to control the flow of the demonstration.
+
+### Elastic Load Balancing
+
+Elastic Load Balancing is used to distribute incoming HTTP traffic across multiple instances.
+This example creates an 
+[Application Load Balancer](https://docs.aws.amazon.com/elasticloadbalancing/latest/application/introduction.html) 
+and adds a listener that forwards requests
+from the load balancer endpoint to the Amazon EC2 instances managed by the Auto Scaling
+group. The target group also performs health checks on the instances and pulls unhealthy
+instances out of the rotation.
+
+## ⚠ Important
+
+* Running this code might result in charges to your AWS account.
+* Running the tests might result in charges to your AWS account.
+* We recommend that you grant your code least privilege. At most, grant only the minimum permissions required to perform the task. For more information, see [Grant least privilege](https://docs.aws.amazon.com/IAM/latest/UserGuide/best-practices.html#grant-least-privilege).
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Scenario
+
+### Prerequisites
+
+To ensure that all of the components used in this example can access one another you must
+create them all in the same VPC. A VPC is a logically isolated network that is provided by
+[Amazon Virtual Private Cloud (Amazon VPC)](https://docs.aws.amazon.com/vpc/latest/userguide/what-is-amazon-vpc.html).
+You can use either the default VPC that is included with your account or create your own.
```

Comment:
Is there a link we could add on how to create your own VPC?

Reply:
Added.